### PR TITLE
Create 2018_indie_mivsy_explination.txt

### PR DESCRIPTION
**Pull request by LowPolyCount**

### CODE REVIEW

File: mivs/templates/emails/2018_indie_mivsy_explination.txt [hunk 1]
```diff
@@ -0,0 +1,35 @@
+Hello Indies,
+
+We here at MIVS are so excited to see you and your games in less than month. Last year, we introduced the MIVSYs, MAGFest’s Indie Awards to be given out to the best of the best. This year, we made a couple changes along with a new addition. We will also be having a small, much less impromptu, awards ceremony on Saturday at 7:00 PM in the MIVS area.  
+
+With that in mind, I would like to take this opportunity to officially awards the awards that will be handed out at this year’s MAGFest.
```
Comment (RobRuana) [inline mivs/templates/emails/2018_indie_mivsy_explination.txt]: Typo perhaps?
> officially awards the awards

Should be?
> officially announce the awards

File: mivs/templates/emails/2018_indie_mivsy_explination.txt [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+Your music is aesthetically pleasing to the ear! This award is given out to the indie game considered to have the best music of all games that were accepted into the Indie Game Showcase. (Note: Some music may be on the loud side. We recommend bringing headphones for attendees so they can properly appreciate the dulcet tones of your game)
+
+-Shiny! Award (Best Art)
+
+Your game is pretty! This award is given out to the indie game considered to have the best artstyle of all games that were accepted into the Indie Game Showcase.
+
+-Tronster Award (Best Booth/Rug)
+
+Cool Booth, Broh! This award is given out to the indie game considered to have the best booth of all games that were accepted into the Indie Game Showcase.. And remember kids, the most important piece when decorating any room is a good rug.
```
Comment (RobRuana) [inline mivs/templates/emails/2018_indie_mivsy_explination.txt]: Were the two periods intentional?
> Indie Game Showcase..

File: mivs/templates/emails/2018_indie_mivsy_explination.txt (hunk 1 above)
Comment (LowPolyCount) [reply mivs/templates/emails/2018_indie_mivsy_explination.txt]: typos should be fixed now.

**Comment by RobRuana** [comment]
Haha, I just noticed that the name of the file is "explination" rather than "explanation". No big deal, as it is spelled the same way in the associated pull request: https://github.com/magfest/mivs/pull/144

You want these merged as-is, or do you wanna fix the filename?

**Comment by LowPolyCount** [comment]
Let's just ship it as it, since it's only for this upcoming year anyway 💃

**Comment by RobRuana** [comment]
Okay! These changes are live on production 😄

**Pull request merged**